backend/src/models/statusUpdate.js

```diff
@@ -0,0 +1,66 @@
+const { Model } = require('objection')
+const tableStatusUpdates = 'statusUpdates'
+
+class StatusUpdate extends Model {
+  static get tableName () {
+    return tableStatusUpdates
+  }
+
+  static get idColumn () {
+    return ['StatusUpdateID']
+  }
+
+  static get relationMappings () {
+    const Student = require('./student')
+    const Status = require('./status')
+
+    return {
+      student: {
+        relation: Model.HasOneRelation,
+        modelClass: Student,
+        join: {
+          from: 'statusUpdates.StudentID',
+          to: 'students.StudentID'
+        }
+      },
+
+      previousstatusID: {
+        relation: Model.ManytoManyRelation,
+        modelClass: Status,
+        join: {
+          from: 'statusUpdates.PreviousStatusID',
+          to: 'statuses.StatusID'
+        }
+      },
+
+      nextstatusID: {
```

Comment:
```suggestion
      nextStatus: {
```
We are retrieving the `Status` at a given `NextStatusID`.